components/financial_assistance/app/controllers/financial_assistance/evidences_controller.rb

```diff
@@ -26,7 +26,7 @@ def update_evidence
     end
 
     def fdsh_hub_request
-      authorize @evidence, :fdsh_hub_request?
+      authorize HbxProfile, :can_fdsh_hub_request?
```

Comment:
Admin has a permission field can_call_hub to request to hub calls.
- replace can_fdsh_hub_request? with can_call_hub?

```suggestion
      authorize HbxProfile, :can_call_hub??
```

Reply:
https://github.com/ideacrew/enroll/pull/3693/commits/9d2dec224cd2d6793b23aef18de83cbe8063b193#diff-5d3bf844d02bcd5391c2833dfca826997d24b37cdb6ad483b324d7039f51d92dL486-L489 && https://github.com/ideacrew/enroll/pull/3693/commits/9d2dec224cd2d6793b23aef18de83cbe8063b193#diff-70153ae0346f1bcff8599f582c3a4ee1febfd1358606de66c3db2ea9406c7256R29